master/config.sls

```diff
@@ -0,0 +1,24 @@
+{% from "master/map.jinja" import master with context %}
+
+include:
+  - .services
```

Comment:
That's why the config states have a `require_in` attribute set to trigger the service reload after they execute.


Regards,

Tobias Macey

Senior DevOps Engineer

MIT Office of Digital Learning

http://odl.mit.edu

On Wednesday, April 20, 2016 8:57:01 AM EDT Brandon DeRosier wrote:

In master/config.sls<https://github.com/mitodl/master-formula/pull/2#discussion_r60436474>:

> @@ -0,0 +1,24 @@

> +{% from "master/map.jinja" import master with context %}

> +

> +include:

> +  - .services

Is it ok that the minion/master services may or may not start after the minion/extra configs are deployed? AFAIK include doesn't guarantee execution ordering on it's own.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/mitodl/master-formula/pull/2/files/596af3f4a2ae3c91315ba42c3df4ec15b26e0d0e#r60436474>